### DISCUSSION TITLE
Wat moet deze week af

### DISCUSSION THREAD

**Discussion by rosa33783** [Sensinki/Blok-Tech-Team-4A#23] [General]
Mijn A1 gedeelte ga ik volgende week af maken. Deze week ga ik me focussen op het team gedeelte en verder met de database.

**Comment by D0nnaz** [comment]
MVC

**Reply by rosa33783** [reply]
Sena

**Comment by D0nnaz** [comment]
Images, audio en handlebars compressen (of iets met grunt of gzip)

**Reply by rosa33783** [reply]
Allemaal

**Comment by D0nnaz** [comment]
Zorgen dat de gebruiker 1x geredirect wordt naar /match

**Reply by rosa33783** [reply]
Sang

**Comment by D0nnaz** [comment]
Zorgen dat de juiste chats in beeld komen

**Reply by rosa33783** [reply]
Sena

**Comment by D0nnaz** [comment]
CSS

**Reply by rosa33783** [reply]
Allemaal

**Comment by D0nnaz** [comment]
Handlebars netjes maken?

**Reply by Sensinki** [reply]
Iedereen

**Comment by D0nnaz** [comment]
Comments plaatsen met uitleg bij eigen code

**Reply by D0nnaz** [reply]
Iedereen

**Comment by D0nnaz** [comment]
Checken of alle code dezelfde standaarden volgt

**Comment by Sensinki** [comment]
Wiki research

**Reply by Sensinki** [reply]
Iedereen

**Comment by D0nnaz** [comment]
Pull request

**Reply by Sensinki** [reply]
Rosalie en Donna

**Comment by Sensinki** [comment]
404 error

**Comment by dinhs-frontend** [comment]
Ik ga deze week verder met "per user naar de juiste pagina sturen als zij geliked hebben, wanneer zij opnieuw ingelogd"

**Comment by Sensinki** [comment]
Style guide

**Comment by D0nnaz** [comment]
Requirement list

**Comment by D0nnaz** [comment]
WIKI eigen delen

**Comment by D0nnaz** [comment]
- [ ] A1 afmaken - Rosalie
- [ ] Pull request - Rosalie 
***
- [ ] Zorgen dat de gebruiker 1x geredirect wordt naar /match - Sang
- [ ]  Compressen images etc. - Sang
***
- [ ] MVC - Sena
- [ ] Zorgen dat de juiste chats in beeld komen - Sena
***
- [ ] Pull request - Donna
- [ ] Style guide - Donna
- [ ] 404 error - Donna
- [ ] MVC - Donna
***
- [ ]  WIKI eigen delen - Iedereen
- [ ] Checken of alle code dezelfde standaarden volgt - Iedereen
- [ ] Comments plaatsen met uitleg bij eigen code - Iedereen
- [ ] Handlebars netjes maken - Iedereen
- [ ] CSS - Iedereen (misschien nog een beetje verdelen)
